addon/ng2/utilities/find-parent-module.ts

```diff
@@ -0,0 +1,34 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+module.exports = function findParentModule(
+  project: any,
+  currentDir: string): string {
+
+  const sourceRoot = path.join(project.root, project.ngConfig.apps[0].root, 'app');
+
+  // trim currentDir
+  currentDir = currentDir.replace(path.join(project.ngConfig.apps[0].root, 'app'), '');
+
+  let parts = currentDir.split(path.sep);
+  const partsLength = parts.length;
+  let pathToCheck = path.join(sourceRoot, currentDir);
+
+
+  for (let i = 0; i <= partsLength; i++) {
```

Comment:
This will go out of the project, which is "dangerous".

`while (pathToCheck.length >= sourceRoot.length) {` or something similar should fix it.